editoast/src/schema/utils/duration.rs

```diff
@@ -117,13 +117,19 @@ where
                     "years and months are not supported",
                 ));
             }
-            ChronoDuration::days(day as i64)
-                + ChronoDuration::hours(hour as i64)
-                + ChronoDuration::minutes(minute as i64)
-                + ChronoDuration::seconds(second as i64)
-                + ChronoDuration::milliseconds(millisecond as i64)
+            ChronoDuration::try_days(day as i64)
+                .ok_or_else(|| serde::de::Error::custom("value for millisecond is not valid"))?
+                + ChronoDuration::try_hours(hour as i64)
+                    .ok_or_else(|| serde::de::Error::custom("value for millisecond is not valid"))?
```

Comment:
The error message should be updated.

```suggestion
                    .ok_or_else(|| serde::de::Error::custom("value for hours is not valid"))?
```